anomaly_agent/nodes.py

```
@@ -0,0 +1,73 @@
+"""Graph nodes for anomaly detection and verification."""
+from typing import Callable, List, Literal, Optional, TypedDict
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)